ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -638,3 +775,27 @@ def get_entity_odds_ratio(event_contingency_table: EventContingencyTable, prior_
         odds_ratio=odds_ratio,
         correlation_type="success" if odds_ratio > 1 else "failure",
     )
+
+
+def build_selector(elements: List[Dict[str, Any]]) -> str:
```

Comment:
https://github.com/PostHog/posthog/blob/master/ee/clickhouse/models/element.py#L20 ? 

Actually, not quite sure if the frontend format turns it back into a string. If so, you can directly use the raw value coming from the DB as well. I think I'll have to write a test to confirm.

Reply:
Yeah I wasn't sure about this bit. I'm not even sure if the selector is required for the people url also, I could just remove the selector altogether?

Reply:
The selector helps accurately filter out the people. We could definitely do with some utilities to make this work uniformly well everywhere. cc: @EDsCODE if you know of something that exists that I might've missed.

For now, happy to not regard for correlations: There's more work to be done for autocapture & persons anyway